src/components/button/button.tsx

```diff
@@ -77,6 +93,7 @@ export class Button {
                     'just-loaded': this.justLoaded && !this.loadingFailed,
                     'just-failed': this.justLoaded && this.loadingFailed,
                     'mdc-button--outlined': this.outlined,
+                    active: this.isActive,
```

Comment:
reminder: we don't need this class in this PR. We would do it in a separate PR, when working on the styles

Reply:
also when we add it, the class should be called `is-active`, not `active` ;)

Reply:
Is it alright if I split the PR into two commits, one for the application logic and one for the styles?

We can create a larger PR where we make this approach more scalable after this gets merged :)

Reply:
No, I need to do some changes for the styles and mixins, or at least be able to review it. Since I'm on vacation, it's better to get this one merged, and we can do next one as soon as I come back. ;)